plugins/chat/assets/javascripts/discourse/components/chat-emoji-picker.hbs

```diff
@@ -15,6 +15,7 @@
     {{will-destroy (if @willDestroy @willDestroy (noop))}}
     {{will-destroy this.removeClickOutsideEventListener}}
     {{on "keydown" this.trapKeyDownEvents}}
+    ...attributes
```

Comment:
is this left over debugging code or some sort of magic I never learned about?

Reply:
made a topic about this